### PR TITLE
split settings into base, develop, production

### LINKED ISSUE #4: Different settings for development and production
I would like to have different settings for development and production.
We shoud have a basic settings file and variants for production, development and staging.

- put settings in a package (folder)
- have sub modules base, develop, production
- manage.py defaults to develop
- wsgi and asgi default to production
- the heroku proc script uses production to prepare db etc.

For development and unit tests it seems much easier for me to user sqlite database. Don't know how to use autogenerated test-db with postgres.
Unfortunately the current model doesn't support sqlite. Will add another issue for that..

### PR DESCRIPTION
**Pull request by samuel-iseli**
implements multiple settings from #4

### CODE REVIEW

File: depotcommun/settings/production.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# Database
+# https://docs.djangoproject.com/en/3.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'depot-commun',
+        'USER': 'postgres',
+        'PASSWORD': 'password',
```
Comment (zuckerruebe) [inline depotcommun/settings/production.py:11]: We'll have to think about removing passwords and so on for production in the future.

Comment (samuel-iseli) [reply depotcommun/settings/production.py:11]: Sure. Passwords and secret key are properties that should be specified on the server. I think heroku has means to do this. That's what the heroku client library that we are using is meant for isn't it?

File: depotcommun/settings/base.py [hunk 2]
```diff
@@ -128,5 +127,3 @@
 
 STATIC_URL = '/static/'
 
-django_heroku.settings(locals())
```
Comment (zuckerruebe) [inline depotcommun/settings/base.py:131]: Should we not also remove the `DATABASES` part in base.py?

Comment (samuel-iseli) [reply depotcommun/settings/base.py:131]: yes that would make it consistent.
will you make the change or should I?

**Pull request merged**